nestor_api/api/api_routes/sample.py

```diff
@@ -0,0 +1,11 @@
+"""Define the sample route."""
+from flask import Blueprint, Response, jsonify
+
+
+def register_routes(api: Blueprint) -> None:
+    """Register the /sample route."""
```

Comment:
```suggestion
    """Register the /sample routes."""
```
The register_routes applies for all sample routes